modules/plot/include/opencv2/plot/plot.hpp

```diff
@@ -0,0 +1,53 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
```

Comment:
the subdirectory plot/ and nested plot.hpp are only needed for older modules. For new ones they are unnecessary and should be deleted